cmd/sentinel/sentinel.go

```diff
@@ -360,6 +360,8 @@ func (s *Sentinel) setDBSpecFromClusterSpec(cd *cluster.ClusterData) {
 			db.Spec.FollowConfig.StandbySettings = clusterSpec.StandbySettings
 		}
 		db.Spec.AdditionalWalSenders = *clusterSpec.AdditionalWalSenders
+		db.Spec.AdditionalSyncStandbyNames = *clusterSpec.AdditionalSyncStandbyNames
```

Comment:
I'll remove this and just add the additional synchronous standbys in db.Spec.SynchronousStandby.